src/swupd/testdata/invalid_manifests/manifest.duplicateVersion

```diff
@@ -0,0 +1,299 @@
+MANIFEST	21
+version:	19440
+version:	19440
```

Comment:
You probably don't need to add such a large file here since it should fail right after reading the header. Probably just a few lines of the body should be sufficient to show the failure is happening because of the header instead of the body.

Reply:
Cropped the test files to be smaller like the existing invalid ones.